llama-index-core/llama_index/core/postprocessor/rankGPT_rerank.py

```diff
@@ -86,6 +86,50 @@ def _postprocess_nodes(
         else:
             return nodes[: self.top_n]
 
+    async def _async_postprocess_nodes(
```

Comment:
we can call this `_apostprocess_nodes` which is the typical convention followed in our library